packages/wallets/src/components/Base/ModalWrapper/ModalWrapper.scss

```diff
@@ -25,3 +17,25 @@
         border-radius: 0;
     }
 }
+.wallets-modal-body {
```

Comment:
are those .css global to all wallet modals, or specific to walletsresetmt5passowrd?

from what I see, they are specific to Walletsresetmt5password, so should be moved to Walletsresetmmt5passwords.scss

Reply:
done...